### ISSUE TITLE
1.4.10: Clarify that two-dimensional layouts must not lose functionality

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag#3151]
Where the two-dimensional layout exception applies, 1.4.10 Reflow allows content to scroll in two dimensions. However, I believe it should not allow loss of information or functionality. Is this correct?

Examples: a data table where the exception applies

* When resized to 200% in a common window width (e.g. 1024 px, yielding a 512 CSS pixel width equivalent), 1.4.4 Resize Text requires that users must be able to bring all of the content into view in some manner, e.g. by horizontal scrolling.
* In a width equivalent of 320 CSS pixels, some of the table content extends off-screen to the right, which is permitted by the two-dimensional layout exception. However, suppose at this smaller width some of the table content can't be scrolled into view or otherwise brought into view. I believe this should fail 1.4.10 because of loss of information or functionality.

**Comment by mitchellevan** [comment]
Relevant: #916

**Comment by maryjom** [comment]
@mitchellevan I agree with this, as when I think even about native software applications where I have increased font size and dialog boxes sometimes extend beyond the bottom of the screen. If they don't allow for scrolling or access to all of the UI components, then not all functionality is available (or may be available by tabbing, but a sighted user can't see what's got focus, but also not clickable by a pointing device). This used to be a problem in Webex screen sharing until an issue was opened and subsequently fixed.

**Comment by detlevhfischer** [comment]
The normative SC text can indeed be read as if the exception for 2-dimensional content would also broadly exempt content from working in terms of information and function. This surely wasn't the intention of the SC. Perhaps an errata or a note should clarify that?

**Comment by scottaohara** [comment]
Figure 13 of the updated Reflow understanding doc touches on this topic - where the failing example shows some of the pagination controls (prev/next buttons) are lost at the smaller viewport size.  

I'm not going to close this issue as resolved though - as i think we could probably create another failing technique to highlight this further.
